src/services/clients/webhooks.client.js

```diff
@@ -81,7 +81,7 @@ async function createWebhook (serviceId, gatewayAccountId, isLive, options = {})
     service_id: serviceId,
     gateway_account_id: gatewayAccountId,
     live: isLive,
-    callback_url: options.callback_url,
+    callback_url: options.callbackUrl || options.callback_url,
```

Comment:
I think for ease of development, specifying callbackUrl, subscriptions and descriptions in the method signature would be preferable to passing in `options`. This means the developer won't have to look into this webhooks.client.js file is order to figure out what the options are.
Furthermore allowing callbackUrl or callback_url here seems a bit messy.

Reply:
I thought camel case was our agreed convention, so that's what I used in the form. But the old settings form uses snake case.  Doing this was the simplest way I could think of making this work without having to refactor the 'old' webhooks settings code. Similarly for the options. It will be easier to make these sorts of improvements once we've deleted the old settings code, won't it?

Reply:
Oh yeah we did agree on camel case. Yeah we refactor this at a later date. Sorry but can you put something like `// TODO refactor to explicitly pass all params in the method once old webhooks controller code is deleted`?